test/v3/controllers/requestParsers/validators/SubmitUkPropertyBsasValidatorSpec.scala

```diff
@@ -16,225 +16,8 @@
 
 package v3.controllers.requestParsers.validators
 
-import play.api.libs.json.Json
-import play.api.mvc.AnyContentAsJson
 import support.UnitSpec
-import v3.fixtures.ukProperty.SubmitUKPropertyBsasRequestBodyFixtures._
-import v3.models.errors._
-import v3.models.request.submitBsas.ukProperty._
 
 class SubmitUkPropertyBsasValidatorSpec extends UnitSpec {
 
-  val bsasId = "a54ba782-5ef4-47f4-ab72-495406665ca9"
```

Comment:
I'm a little confused by this, surely there still needs to be tests here? or is this removing the tests so whoever does parser and validator can write new ones?